src/box/sql/select.c

```diff
@@ -2209,6 +2209,7 @@ sqlColumnsFromExprList(Parse * parse, ExprList * expr_list,
 		}
 		memcpy(space_def->fields[i].name, zName, nName);
 		space_def->fields[i].name[nName] = '\0';
+		sqlDbFree(db, zName);
```

Comment:
Above we use `sql_free` to free `zName`. Shouldn't we use the same function in both places?